src/main/scala/is/hail/methods/VariantQC.scala

```diff
@@ -46,113 +29,88 @@ object VariantQCCombiner {
     "rHetHomVar" -> TFloat64,
     "rExpectedHetFrequency" -> TFloat64,
     "pHWE" -> TFloat64)
-}
-
-class VariantQCCombiner extends Serializable {
```

Comment:
The code below is a bit monolithic.  I like this class.  Option: leave it in, with `mergeGT(gt: Int)`, `mergeDP`, etc.